Cargo.toml

```diff
@@ -35,6 +36,8 @@ num-traits = { version = "^0.2.14", optional = true }
 rand = { version = "^0.8.4", optional = true }
 sha2 = { version = "0.10.2", optional = true }
 rsa = { version = "^0.5.0", optional = true }
+iocuddle = { version = "0.1.1", optional = true }
+libc = { version = "0.2", optional = true }
```

Comment:
If we were keeping `libc` as a dependency (we aren't: see below), I'd suggest moving this to `dev-dependencies`.